src/lib/onboard/managed-bootstrap/docker-journal.ts

```
@@ -0,0 +1,389 @@
+      fs.unlinkSync(temporary);
+    } else {
+      fs.renameSync(temporary, target);
+    }
+    fs.chmodSync(target, JOURNAL_FILE_MODE);
+    fsyncDirectory(directory);
+  } finally {
+    if (descriptor !== null) fs.closeSync(descriptor);
+    try {
+      fs.unlinkSync(temporary);
+    } catch (error) {
+      if ((error as NodeJS.ErrnoException).code !== "ENOENT") throw error;
```

Biome (2.5.5)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)